Doc/library/subprocess.rst

```diff
@@ -625,6 +625,13 @@ functions.
       * :data:`CREATE_DEFAULT_ERROR_MODE`
       * :data:`CREATE_BREAKAWAY_FROM_JOB`
 
+   *pipesize* can be used to change the size of the pipe when
+   :data:`PIPE` is used for *stdin*, *stdout* or *stderr*. The size of the pipe
+   is only changed on platforms that support this.
```

Comment:
mention something like (Linux at the time of this writing) at the end here.

that's enough of a documentation carrot to cause someone who wants this on Windows, macOS, or some BSD to pop up and contribute support on that platform in the future.